.github/workflows/format.yml

```
@@ -0,0 +1,33 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Format CI
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the development branch
+  [push, pull_request]
+  
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+          fetch-depth: 0
+
+      - name: Setup Node.js environment
+        uses: actions/setup-node@v2.1.4
+        with:
+          node-version: '12'
+      
+      # Format Checking
+      - run: git fetch --no-tags --prune --depth=5 origin ${{ github.event.pull_request.base.ref }}
+      - run: npm ci
+      - run: npm run format:check -- --base=origin/${{ github.event.pull_request.base.ref }}.
```

There is an extra dot at the end of the command causing it to fail
```suggestion
      - run: npm run format:check -- --base=origin/${{ github.event.pull_request.base.ref }}
```